gamma/1.4/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# base image
+FROM ubuntu:bionic
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL container.version="1"
+LABEL software="centroid"
+LABEL software.version="v1.4"
+LABEL description="Gene Allele Mutation Microbial Assessment"
+LABEL website="https://github.com/rastanton/GAMMA/tree/v1.4"
+LABEL license=""
+LABEL maintainer="Rachael St. Jacques"
+LABEL maintainer.email="rachael.stjacques@dgs.virginia.gov"
+
+# install dependencies
+RUN apt-get update && apt-get -y --no-install-recommends install \
+    unzip \
+    tar \
+    python3.7 \
+    python3-pip \
+    gzip \
+    wget \
+    git \
+    curl \
+    make \
+    g++ \
+    ncbi-blast+ \
+    hmmer && \
+    apt-get autoclean && \
+    rm -rf /var/lib/apt/lists/*
+
+#install biopython
+RUN pip3 install biopython cgecore
```

Comment:
I would recommend combining all of these `pip3 install` layers into one, unless there is some reason not to that I'm unaware of.

`RUN pip3 install biopython cgecore tabulate` should install all at once.